### PR TITLE
docs: add Chinese documents

### PR DESCRIPTION
**Pull request by dohooo**
#32

### CODE REVIEW

**Comment by dohooo** [comment]
[Preview](https://github.com/dohooo/vue-starport)

File: README.md [hunk 1]
```diff
@@ -71,12 +72,13 @@ At root `App.vue`, add `<StarportCarrier>` component from `vue-starport` at the
 
 ```html
 <script setup>
-import { StarportCarrier } from 'vue-starport'
+  import { StarportCarrier } from "vue-starport";
```
Comment (antfu) [inline README.md:75]: Please revert the format changes

**Review by antfu — CHANGES_REQUESTED** [review]

**Comment by antfu** [comment]
You can use [tldraw](https://www.tldraw.com/) to open the source file https://github.com/antfu/vue-starport/blob/main/graphs/source.tldr and generate the new image.

**Comment by dohooo** [comment]
> You can use [tldraw](https://www.tldraw.com/) to open the source file https://github.com/antfu/vue-starport/blob/main/graphs/source.tldr and generate the new image.

image

There are some problems with Chinese export..

**Comment by dohooo** [comment]
You can [recheck it](https://github.com/dohooo/vue-starport/blob/main/README.zh-Hans.md). I took screen short.

**Comment by dohooo** [comment]
Hi @antfu , Do I need to do anything else?

**Pull request merged**